library/general/requirednamespacequotas/samples/constraint.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sRequiredNamespaceQuota
+metadata:
+  name: enforce-namespace-quota
+spec:
+  match:
+    kinds:
+      - apiGroups: [""]
+        kinds: ["Namespace"]
+  enforcementAction: "warn"
```

Comment:
This constraint needs to be enforced only for "audit" since `ResourceQuota` will not be present at admission time for `Namespace`. Find out more about `enforcementPoints` here - https://open-policy-agent.github.io/gatekeeper/website/docs/enforcement-points/#only-audit.